tests/scene/test_code_edit.h

```diff
@@ -3446,7 +3446,7 @@ TEST_CASE("[SceneTree][CodeEdit] completion") {
 	}
 
 	SUBCASE("[CodeEdit] autocomplete suggestion order") {
-		/* Favorize less fragmented suggestion. */
+		/* Prefer less fragmented suggestion. */
```

Comment:
```suggestion
		// Prefer less fragmented suggestion.
```
Same everywhere here

Reply:
This might be better in a separate PR as it is pretty inconsistent across this whole file. If need be I can still make this modification.

Reply:
Meant all the ones you just changed